src/api/app/views/webui/request/_lock_unlock_comment.html.haml

```diff
@@ -0,0 +1,8 @@
+%li.nav-item
+  = link_to('#', class: 'nav-link', data: { 'bs-toggle': 'modal', 'bs-target': "#locking-request-#{bs_request.id}-modal" }) do
+    - if bs_request.comment_lock.blank?
+      %i.fa.fa-lock
```

Comment:
```suggestion
      %i.fas.fa-comment-slash
```
It might be nicer to use stacked icons instead, but if you don't want to experiment with them, this is a big improvement already

Reply:
Sure. Applied.